### DISCUSSION TITLE
useRequest 的轮询模式下的 runAsync 是怎么使用的

### DISCUSSION THREAD

**Discussion by ZhihaoLau** [alibaba/hooks#2378] [Q&A] [answered]
useRequest 的轮询模式下的 runAsync 定义是(...params: TParams) => Promise<TData>; 轮询是请求多次的, 而Promise只会被 resolve 一次, 请问这个API的作用是什么

**Comment by liuyib** [comment, marked as answer]
官网文档还没更新，所以我把 pr 截图给你吧：
https://github.com/alibaba/hooks/pull/2358/files
![image](https://github.com/alibaba/hooks/assets/38221479/4f05dd1e-e177-4a43-b6fc-7e702e683ba4)

轮询时一般不会直接使用 runAsync，除非需要手动触发请求了，可以用 run或者runAsync
